internal/rotate.go

```diff
@@ -46,13 +50,27 @@ type CertRotationStep interface {
 
 // NewCertRotationHandler constructs a new handler to initiate a rotation with
 func NewCertRotationHandler(app *App, stateFile, estServer string) *CertRotationHandler {
+	corId := fmt.Sprintf("certs-%d", time.Now().Unix())
```

Comment:
Maybe it will be useful to include `RotationId` into `corId` or maybe they should be equal?

Reply:
I thought a lot about having RotationId derive or be == to corId. The thing I don't like about it is that you wind up trusting user input to be something usable on our backend. In theory - RotationId will come from something else we've generated, I just wasn't sure if it should be the value.

Reply:
ok, but rotation ID is generated by `fioctl` isn't it.
It's not critical, I just thought it would be nice to have something that allows mapping an event cor ID to a rotation ID. Since we we don't allow many simultaneous rotation then rotation ID and cor IDs can be mapped through timing.

Reply:
we can use the incoming rotationid. There's a very small chance someone will do this using their own calls to our API that could set rotation id to something we don't know about. But it doesn't cause bugs - just makes the update events confusing.

There's a really nice feature by using it you mention - you can better correlate things.

Reply:
my first implementation used rotationId and i figured you guys were going to reject it, so i switched to this approach